src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
+
+  // Поділ  рядків крапками з комою
```

Comment:
If you add comments, it is better in English.
Also, comments should be moderate. You don't just have to write comments on everything.